tests/unit/AnnualReport.spec.ts

```diff
@@ -869,12 +867,16 @@ describe('Annual Report - Part 3 - Submitting', () => {
     sinon.restore()
   })
 
-  xit('saves a new filing and redirects to Pay URL when this is a new AR and the File & Pay button ' +
+  it('saves a new filing and redirects to Pay URL when this is a new AR and the File & Pay button ' +
     'is clicked', async () => {
+    // set configurations
+    const configuration = {
+      'VUE_APP_AUTH_WEB_URL': 'https://auth.web.url/'
+    }
+    configurationStore.setConfiguration(configuration)
```

Comment:
It's probably a good idea to leave this at the top of the test suite, even if a specific test doesn't need this. (Ie, initialize test environment once)

However, I'd move it to `beforeAll()` so it's done just once instead of for every test.

What do you think?

Reply:
I thought of it but didn't do it somehow :). I think moving it in `beforeAll()` makes sense.

Reply:
Done, thanks.